news/2022-08-30-changes-to-osu-sr-and-pp.md

```diff
@@ -0,0 +1,98 @@
+
+The number of notes that are considered difficult by Speed calculations will now be counted. Accuracy will then be scaled by the average between this count and the overall object count. This means that for those maps with filler, the punishment from 100s/50s/misses will now be more accurate. The averaging is done for the sake of leniency - using the Speed count alone resulted in values that were too harsh in testing.
+
+### Remove non-overlapping velocity change bonus from Aim
+
+Throughout the past few months, [Lasse's Zan'ei [Illusion]](https://osu.ppy.sh/beatmapsets/1759729#osu/3601629) has had widespread community attention thanks to it's abuse of the newly introduced velocity change buff. Coupled together with the wide angle bonus, the result is a notoriously overweight map that has found it's way into ~31% of all 5 digit player's top plays.
+
+Fixing this was a high priority task, so [this proposal](https://github.com/ppy/osu/pull/19004) to remove the non-overlapping segment of the velocity change buff was approved. This fixes maps such as Zan'ei, but it unfortunately nerfs maps that genuinely benefited from the buff such as [eiri-'s Trance Dance Anarchy [Don't Look Back]](https://osu.ppy.sh/beatmapsets/1124084#osu/2348869). This is an unfortunate but necessary side effect, and rebuffing these maps is considered high priority in the future.
+
+Maps featuring cutstreams are mostly unaffected.
+
+### General balancing pass
+
+A general balancing pass, mostly for performance points calculation, was applied in [this proposal](https://github.com/ppy/osu/pull/19001) by [StanR](https://osu.ppy.sh/users/7217455). This aims to alleviate some concerns shared by both the team of developers and the community.
+
+- **Aim has been slightly buffed**. Slider aim and high BPM aim should be mostly unaffected by this.
+- **Speed's 50s nerf has been reduced.** Something highly requested by the Speed community and now doable with worst-case accuracy speed scaling.
+- **Low AR multiplier has been reduced.** There were cases where maps were worth more with HTEZ than HT alone.
+
+### Improved double tap detection in Speed
+
+It's extremely rare that maps need to be disqualified and put on hold from getting to the Ranked section because of pp abuse, however push came to shove a few months ago when [pewdekz's POSSESSION mapset](https://osu.ppy.sh/beatmapsets/1691083#osu/3455732) was [held back forcefully by NAT](https://osu.ppy.sh/beatmapsets/1691083/discussion/-/generalAll#/3020552). The map features incredibly quick doubles that can be easily doubletapped, and as consequence of the Speed cap removal in the previous group of changes, they were considered much harder than they were. This is now due to improve, as per [this change](https://github.com/ppy/osu/pull/18692) by [Apo11o](https://osu.ppy.sh/users/9558549)!
```

Comment:
```suggestion
It's extremely rare that maps need to be disqualified and put on hold from getting to the Ranked section because of pp abuse, however push came to shove a few months ago when [pewdekz's POSSESSION mapset](https://osu.ppy.sh/beatmapsets/1691083#osu/3455732) was [held back forcefully by the NAT](https://osu.ppy.sh/beatmapsets/1691083/discussion/-/generalAll#/3020552). The map features incredibly quick doubles that can be easily doubletapped, and as a consequence of the Speed cap removal in the previous group of changes, they were considered much harder than they actually were. This is now due to improve, as per [this change](https://github.com/ppy/osu/pull/18692) by [Apo11o](https://osu.ppy.sh/users/9558549)!
```

Reply:
Wasn't there some discourse as to whether `the NAT` vs. `NAT` is correct? Was the former decided to be more correct?

Reply:
yes there has been a [few](https://github.com/ppy/osu-wiki/pull/7245#discussion_r857820020) [discussions](https://discord.com/channels/188630481301012481/218677502141399041/971128809036329031) about whether to use "a" or "an" for things like "NAT member" (which imo should be "an"), but it doesn't apply here. "the NAT" is definitely correct in this case.